speaker_verification/sql_utils.py

```diff
@@ -130,14 +144,21 @@ def insert_db_row(table, id, mfcc):
     ----------
     table : str
         Name of table to insert record within.
-    id : str
+    id : int
         Id key for required record within table for insertion.
     mfcc : numpy.array
         MFCC dataset to be inserted within database records.
     """
-    try:
-        with sqlite3.connect(DATABASE_PATH, detect_types=sqlite3.PARSE_DECLTYPES) as con:
-            cur = con.cursor()
-            cur.execute(f"insert into {table}(id, arr) values (?, ?)", (id, mfcc,))
-    except Exception as err:
-        print("Database doesn't exist: ", err)
+    length = len(str(id))
+    if length==9:
+        if (isinstance(table,str) and isinstance(id,int) ):
```

Comment:
Instead we could refactor the method parameters to cast to these types like:
```python
def insert_db_row(table: str, id: int, mfcc):
```
This way we don't have to raise a `TypeError`.

Reply:
I think if the parameter is set, then other functions' parameters should be typed in order to keep consistency in writing code.

Reply:
Yes, but this PR's scope is only concerned with the unit tests for `sql_utils.py` and the unit tests. We can create a new issue to track which functions should have parameter annotations for type checking.